exchanges/bybit/bybit_wrapper.go

```diff
@@ -2322,24 +2369,41 @@ func (by *Bybit) GetFuturesContractDetails(ctx context.Context, item asset.Item)
 			}
 			contractType := strings.ToLower(instruments[i].ContractType)
 			var s, e time.Time
-			s = time.UnixMilli(int64(instruments[i].LaunchTime.Float64()))
-			e = time.UnixMilli(int64(instruments[i].DeliveryTime.Float64()))
+			if instruments[i].LaunchTime > 0 {
+				s = time.UnixMilli(int64(instruments[i].LaunchTime.Float64()))
+			}
+			if instruments[i].DeliveryTime > 0 {
+				e = time.UnixMilli(int64(instruments[i].DeliveryTime.Float64()))
+			}
 
 			var ct futures.ContractType
-			if contractType == "linearperpetual" {
+			switch contractType {
+			case "linearperpetual":
 				ct = futures.Perpetual
-			} else if contractType == "linearfutures" {
+			case "linearfutures":
```

Comment:
For some reason this never gets hit and we are missing 14 linearFutures contracts e.g.
```
{"symbol":"BTC-29MAR24","contractType":"LinearFutures","status":"Trading","baseCoin":"BTC","quoteCoin":"USD","launchTime":"1688716800000","deliveryTime":"1711699200000","deliveryFeeRate":"0","priceScale":"2","leverageFilter":{"minLeverage":"1","maxLeverage":"50.00","leverageStep":"0.01"},"priceFilter":{"minPrice":"0.50","maxPrice":"999999.00","tickSize":"0.50"},"lotSizeFilter":{"maxOrderQty":"10.000","minOrderQty":"0.001","qtyStep":"0.001","postOnlyMaxOrderQty":"50.000"},"unifiedMarginTrade":true,"fundingInterval":480,"settleCoin":"USDC","copyTrading":"none"}
```